README.md

```diff
@@ -1 +1,124 @@
-# .github
\ No newline at end of file
+
+# Repositories: 
+- Service
+- Purpose
+- Language
+- External Integration
+- Database
+- Infrastructure
+
+
+## org: ubiquibot
+
+### 1. ubiquibot/conversation-rewards
+- Service: Conversation Reward Module
+- Purpuse: This is intended to be the proper implementation of comment incentives, based on our learnings from the first go-around.
+- Language: TypeScript
+- External Integration: Github Action, Github API
+- Database: None
+
+
+### 2. ubiquibot/permit-generation
+- Service: Permit Generation module
+- Purpuse: The purpose of this module is to isolate the bot's payment capabilities. This can be directly invoked from other plugins
+- Language: TypeScript
+- External Integration: ENS
+- Database: Supabase
+
+### 3. ubiquibot/assistive-pricing
+- Service: Assistive Pricing module
+- Purpose: **Don't know**
+- External Integration: 
+- Database: Supabase
+
+### 4. ubiquibot/comment-incentives
+
+**Deprecated in favor of @ubiquibot/conversation-rewards**
+
+- Service: Comment Incentives module
+- Purpose: Split reward to the collaborators who wrote some comments
+- Language: Typescript
+- External Integration: 
+- Database: None (Supabase, TODO: get the assignee multiplier from the database)
+
+### 5. ubiquibot/plugins-wishlist
+- Service: not a service
+- Purpose: Wishlist of the future plugins
+
+### 6. ubiquibot/configuration
+- Service: not a service
+- Purpose: Helpers to parse and merge configuration files for Ubiquibot.
+
+### 5. ubiquibot/e2e-tests
+- Service: not a service
+- Purpose: Use this for end-to-end tests.
+@pavlovcik How does it work?
```

Comment:
This was supposed to be used as a dedicated testing sandbox essentially. On the old bot there was a test command and it would run through all of its commands on that repository and read the results and compare them. A true end-to-end test environment. Unfortunately we haven't used or discussed this in a long time, it could be considered deprecated.